WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -133,7 +135,8 @@
           <Border Classes="warning">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction uses semi-private coins." />
+              <TextBlock Text="Transaction uses semi-private coins." 
+                         ToolTip.Tip="This transaction might not be private enough for your needs because when multiple coins are used for a transaction, the privacy of it is only as good as the lowest privacy of the coins used.  You should conjoin first to have (enough) private coins for this transaction." />
```

Comment:
A shorter version of Turbolay's suggestion:

```suggestion
                         ToolTip.Tip="This transaction is not private enough because its privacy is as good as the used coins' lowest privacy. Coinjoin more to have (enough) private coins for this transaction." />
```